src/renderer/routes/test_sequencer_panel/utils/SequenceHandler.ts

```diff
@@ -110,6 +120,12 @@ export async function saveSequences(
     if (res.isErr()) {
       return err(res.error);
     }
+    stateManager.setSequences(
+      [...stateManager.sequences].map((seq) => ({
```

Comment:
no need to spread (?)

Reply:
True, but I realized that the change is not reflected right away in the UI :/